indico/modules/events/client/js/reviewing/components/ReviewForm.jsx

```diff
@@ -71,41 +62,40 @@ export default function ReviewForm() {
               <div className="f-self-stretch">
                 <FinalForm
                   onSubmit={() => {}}
-                  initialValues={{comment: '', visibility: 'editors'}}
-                  subscription={{submitting: true}}
+                  initialValues={{comment: '', protected: false}}
+                  subscription={{submitting: true, values: true}}
                 >
                   {fprops => (
                     <Form onSubmit={fprops.handleSubmit}>
-                      <FinalTextArea
+                      <InputComponent
+                        {...inputProps}
                         onFocus={onCommentClickHandler}
                         name="comment"
-                        rows={commentFormVisible ? 3 : 1}
                         placeholder={Translate.string('Leave a comment...')}
-                        style={{resize: commentFormVisible ? 'vertical' : 'none'}}
                         hideValidationError
                         required
                       />
                       {commentFormVisible && (
-                        <>
-                          <FinalDropdown
-                            name="visibility"
-                            options={visibilityOptions}
-                            width={8}
-                            selection
-                            required
+                        <FinalCheckbox
+                          label={Translate.string(
+                            'Restrict visibility of this comment to other editors'
+                          )}
+                          name="protected"
+                          toggle
```

Comment:
hm, wouldn't an actual checkbox be nicer?

Reply:
I've suggested a toggle :) What others think? @panagiotappl @pferreir

Reply:
To be honest, I prefer a checkbox a bit more since it is a bit more obvious what it means when I check it off.

Reply:
But let's see what others think.